coalib/bears/Bear.py

```diff
@@ -16,14 +16,31 @@
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.output.printers.LogPrinter import LogPrinterMixin
 from coalib.results.Result import Result
+from coalib.misc.Constants import FALSE_STRINGS, TRUE_STRINGS
 from coalib.results.TextPosition import ZeroOffsetError
 from coalib.settings.FunctionMetadata import FunctionMetadata
 from coalib.settings.Section import Section
 from coalib.settings.ConfigurationGathering import get_config_directory
+from coalib.settings.Setting import typed_list
 
 from .meta import bearclass
 
 
+def _debugger_helper(bear):
+    if bear is None:
+        return False
+    elif 'debug_bears' not in bear.section or (
+            bear.section['debug_bears'].value.lower() in FALSE_STRINGS):
```

Comment:
`bool(bear.section['debug_bears'])`

Reply:
Please check I am following the previous instructions also for the value like `na` or `non` etc this will raise ValueError

Reply:
Please test what I suggested before saying it will raise an exception .

And see `StringConverter.__bool__` for why it *should* work.
If it doesnt work, it is because there is something else wrong with the `debug_bears` Setting.

Reply:
Check this https://bpaste.net/show/1faff582d756

Reply:
```
>>> section.append(Setting('abc', 'nop'))
>>> bool(section['abc'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/coala_utils/string_processing/StringConverter.py", line 46, in __bool__
    raise ValueError
ValueError
>>> 
```

Reply:
Actually, I've seen that this is kinda buggy because we don't use coala's `FALSE_STRINGS`, but `coala_util`'s `FALSE_STRINGS`, which isn't so comprehensive than coala's list. However, @jayvdb is right to use `bool` on `Section`, it works in principle and that's what our section system uses.